kubernetes/kustomization.yaml

```diff
@@ -0,0 +1,19 @@
+commonLabels:
+  app.kubernetes.io/name: kube-state-metrics
+
+imageTags:
+- name: quay.io/coreos/kube-state-metrics
+  newTag: v1.5.0
```

Comment:
Do these actually need to be specified here? I'd prefer if we don't have more places we need to update these at.

Reply:
They're not necessary; I'll delete 'em.